docs/applications/add-ssl-certificate.md

```diff
@@ -13,6 +13,24 @@ links:
     featured:
 ---
 
+Before issuing an Secure Sockets Layer (SSL) certificate, a valid Domain Name System (DNS) record must be pointing to the server on which the application is deployed, so the SSL Certificate can be issued and validated for the configured application web domain.
```

Comment:
Not sure if really needed, but perhaps an example would be helpful here.
Please have a look and see if it makes sense to you:

```suggestion
Before an SSL certificate can be issued, a valid DNS record must point to the server on which the application is deployed, so the SSL Certificate can be issued and validated for the configured application web domain. For example, if your application is publicly accessible through the domain `my-application.example.com` the subdomain `my-application` on the domain `example.com` must be configured to point to your server IP address.
```

Reply:
Agreed